bootstrap/eksctl/README.md

```diff
@@ -37,6 +37,8 @@ ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
 
 OIDC_PROVIDER=$(aws eks describe-cluster --name crossplane-blueprints --query "cluster.identity.oidc.issuer" --output text | sed -e "s/^https:\/\///")
 
+VPC_ID=$(aws eks describe-cluster --name crossplane-blueprints --query "cluster.resourcesVpcConfig.vpcId" --output text)
```

Comment:
`VPC_ID` is not used in this section. Let's move it to where it's actually used.

Reply:
change made.